### ISSUE TITLE
Licensing Rationale

### ISSUE THREAD

**Issue by Rowansiv** [UBC-MDS/EDAhelper#78]
Issue to discuss licensing and rationale

**Comment by Rowansiv** [comment]
I suggest we use the MIT license which we currently have.

Given that we have built this package as part of coursework and the aim is to assist others in simplifying steps in EDA, the MIT license would allow the code to be freely distributed for public use.  Furthermore, in our presentation we recognise that only 4 weeks of work was put into the package and is very basic , as such users may want to modify and alter the code to their specific needs when using the package.  By utilising the MIT package, we could gain insight and ideas from others who which to collaborate and help us improve this package in the future.

A 'drawback' of this license is that we would be forfeiting monetary returns if this is used to develop third party software but given that this isn't a priority for our group this point is moot.

I believe that overall our group should use the MIT license as it is the most permissive and user friendly which is what we are aiming for by creating the EDAhelper package.

**Comment by stevenleung2018** [comment]
I agree that we should use MIT license for the reasons given. Thanks.

**Comment by jennifer-hoang** [comment]
I agree with the comments above. Thanks!
